tests/wicked/advanced_sut.pm

```diff
@@ -21,11 +22,28 @@ use utils 'systemctl';
 use lockapi;
 use mmapi;
 
+sub save_test_result {
+    my ($ip, $ip_version, %results) = @_;
+    my $timeout = "60";
+    my $ret = $self->ping_with_timeout(ip => "$ip", timeout => "$timeout", ip_version => $ip_version);
+    if (!$ret) {
+        record_info("Can't ping IP $ip", result => 'fail');
+        $results{'Test1'} = "FAILED";
+    }
+    else {
+        $results{'Test1'} = "PASSED";
+    }
+}
+
 sub run {
     my ($self) = @_;
-    record_info('Test 1', 'Create a gre interface from legacy ifcfg files');
+    my %results;
+    my $iface      = script_output('ls /sys/class/net/ | grep -v lo | head -1');
     my $gre_config = '/etc/sysconfig/network/ifcfg-gre1';
+    my $sit_config = '/etc/sysconfig/network/ifcfg-sit1';
     $self->get_from_data('wicked/ifcfg-gre1_', $gre_config, add_suffix => 1);
+    $self->get_from_data('wicked/ifcfg-sit1_', $sit_config, add_suffix => 1);
```

Comment:
if we copy this file without replacing the dummy values ('local_ip', 'remote_ip', 'tunnel_ip') and we do an ifup, wicked will complain. So, I would move this line to the corresponding test case block.

See this: http://fromm.arch.suse.de/tests/677/file/serial_terminal.txt
    ifup gre1 ; echo RvmM0 -$?-
    wicked: ifcfg-sit1: Cannot parse TUNNEL_LOCAL_IPADDR="local_ip"
    gre1            up